q2_emperor/_plot.py

```diff
@@ -71,17 +73,25 @@ def plot(output_dir: str, pcoa: skbio.OrdinationResults,
 
     generic_plot(output_dir, master=pcoa, metadata=metadata, other_pcoa=None,
                  ignore_missing_samples=ignore_missing_samples,
-                 custom_axes=custom_axes, plot_name='plot')
+                 custom_axes=custom_axes)
 
 
 def procrustes_plot(output_dir: str, reference_pcoa: skbio.OrdinationResults,
                     other_pcoa: skbio.OrdinationResults,
-                    metadata: qiime2.Metadata, custom_axes: str = None,
+                    metadata: qiime2.Metadata, m2_stats: pd.DataFrame = None,
+                    custom_axes: str = None,
                     ignore_missing_samples: bool = False) -> None:
+    info = None
+    if m2_stats is not None:
+        m2 = round(m2_stats['Procrustes Results'][0], 5)
+        trials = m2_stats['Procrustes Results'][2]
+        dec_places = math.floor(math.log10(trials))
```

Comment:
I think `ceil` instead of `floor` would be better here, right?:

```suggestion
        dec_places = math.ceil(math.log10(trials))
```